lib/elixir_boilerplate_web/layouts/templates/flash.html.heex

```diff
@@ -0,0 +1,10 @@
+<% id = "flash-" <> to_string(@kind) %>
```

Comment:
I was under the impression defining variable match inside of heex template was breaking the static/dynamic checks in the live view stateful process?